src/distributedTracingManager.ts

```diff
@@ -43,7 +43,6 @@ export class DistributedTracingManager extends BaseExplorer {
             return;
         }
 
-        this._outputChannel.show();
         await this.updateDistributedTracingSettingForDevices(deviceIds, iotHubConnectionString, updateType, node);
```

Comment:
What if deviceIds.length === 1?

Reply:
If devices.lenght ===1 then we will not use jobs